### PR TITLE
Events api merge

### PR DESCRIPTION
**Pull request by vinhnguyen0429**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -11,22 +11,25 @@
...
+require_once('inc/utilities.php');              // Utility functions
+require_once('inc/get-posts-rest-api.php');     // Pulls posts through the REST API
+require_once('inc/get-campaign-posts-api.php'); // Pulls campaign posts through the REST API
+require_once('inc/get-news-rest-api.php');      // Pulls news through the REST API
+require_once('inc/get-campaign-news-api.php');  // Pulls campaign news through the REST API
+require_once('inc/get-region-news-api.php');    // Pulls region news through the REST API
+require_once('inc/get-events-api.php');         // Pulls events api
+require_once('inc/pagination.php');             // Pagination function
 require_once('inc/comments.php');
+require_once('inc/add-eventsdates-rest-api.php');
```
Comment (brown-a2) [inline functions.php]: Comments need to be added and maybe group with the other events api require?

File: inc/add-eventsdates-rest-api.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        'tax_query' => array(
+            array(
+                'taxonomy' => 'agency',
+                'field' => 'term_id',
+                'terms' => $agency,
+            )
+        )
+    );
+
+    //print_r(new WP_Query());
```
Comment (brown-a2) [inline inc/add-eventsdates-rest-api.php]: Remove all print_r() now?

File: inc/search.php [hunk 3]
```diff
@@ -5,6 +5,122 @@
...
+
+            $start_time = $post['event_start_time'];
+            $end_time = $post['event-end_time'];
+
+            $location = $post['event_location'];
+            
+                if ($datevalueselected === $strip_start_date){
+                    
+                    ?>
+                    <article class="c-events-item" data-type="event">
```
Comment (brown-a2) [inline inc/search.php]: Do you think we should be using the c-calendar component here? rather then recopying the code?

Comment (vinhnguyen0429) [reply inc/search.php]: yup agreed. i'll add it in

File: inc/add-eventsdates-rest-api.php (hunk 2 above)
Comment (vinhnguyen0429) [reply inc/add-eventsdates-rest-api.php]: cool. will do

File: functions.php (hunk 1 above)
Comment (vinhnguyen0429) [reply functions.php]: sure.

File: src/globals/js/blog-content_filter.js [hunk 4]
```diff
@@ -111,4 +111,54 @@ jQuery(function ($) {
...
+                valueSelected: valueSelected,
+                postType: postType
+            },
+            success: function (response) {
+                $('.c-article-item').remove();
+                $content.html(response);
+            }
+        });
+
+        // jQuery.ajax({
```
Comment (brown-a2) [inline src/globals/js/blog-content_filter.js]: Is this needed anymore?

File: inc/get-events-api.php [hunk 5]
```diff
@@ -46,11 +45,7 @@ function get_events_api() {
             
             ?>
                 <article class="c-events-item" data-type="event">  
-                    <time class="c-calendar-icon" datetime="<?php echo $start_date . ' ' . $start_time; ?>">
-                        <span class="c-calendar-icon--dow"><?php echo $get_day; ?></span>
-                        <span class="c-calendar-icon--dom"><?php echo $get_day_num; ?></span>
-                        <span class="c-calendar-icon--my"><?php echo $get_month . ' ' . $get_year; ?></span>
-                    </time>
+                    <?php include(locate_template('src/components/c-calendar-icon/view-eventsapi.php')); ?>
```
Comment (brown-a2) [inline inc/get-events-api.php]: there is an easier way to do this using get_template_part('componet path/view', 'eventsapi' ) but this works fine as well!

**Pull request merged**